readme.md

```diff
@@ -638,7 +633,7 @@ public async void ProductDoesNotExist()
             .WithStatus(HttpStatusCode.NotFound);
 
     await pact.VerifyAsync(async ctx => {
-        var response = await ApiClient.GetProduct(11);
+        var response = await new ApiClient(ctx.MockServerUri).GetProduct(11);
         Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
```

Comment:
consumer expects a `404` here, but the new API provider returns a `200` where products are not found